### PR TITLE
update installation instruction

### PR DESCRIPTION
**Pull request by alberttwong**
install chalk and playwright

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Enter robots! Our automated friends that exist to perform exactly this kind of m
 
 ```sh
 npm i -g california-license-plates
+npm i chalk@4.1.2
```
Comment (zeke) [inline README.md:25]: Hi thanks for the PR.

If these are needed, they should be added to the package.json file, right?

Comment (alberttwong) [reply README.md:25]: They should but I just added more instructions.   I'm not a node guy so I don't know about the npx statement.

Comment (zeke) [reply README.md:25]: Hey I went ahead and updated it myself in https://github.com/zeke/california-license-plates/commit/a73ade01de49c7105c5e8e7a925c02791b03545f


Thanks for inspiring me to revive this project! I made a little video about it here: https://x.com/zeke/status/1834049560815317277

**Comment by alberttwong** [comment]
Awesome. I think it's a great project!

**Pull request closed without merging**